spec/factories/users.rb

```diff
@@ -21,6 +21,14 @@
     trait :administrator do
       association :permission, factory: :administrator_permission
     end
+
+    trait :system do
+      first_name "GoodCity"
+      last_name  "Team"
+      mobile     SYSTEM_USER_MOBILE
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.